africanus/calibration/tests/test_utils.py

```diff
@@ -26,65 +21,83 @@ def give_lm(n_dir):
     return lm
 
 
-def give_flux(n_dir):
-    return 0.1 + np.random.random(n_dir)
+def give_flux(n_dir, n_chan, cor_shape, alpha, freq, freq0):
```

Comment:
This is a https://en.wikipedia.org/wiki/Factory_method_pattern

```suggestion
def flux_factory(n_dir, n_chan, cor_shape, alpha, freq, freq0):
```